### PR TITLE
fixing links to mentoring guild (issue #27)

### LINKED ISSUE #27: The links are still pointing to HackMD
@melissawm I realised the 2 links on [our project](https://numfocus.github.io/disc-unconference-2023-projects/en/mentoring_guide_projects.html) - [Checklist for mentee onboarding](https://hackmd.io/@melissawm/onboarding) and [Agenda template for 1-on-1 meetings](https://hackmd.io/@melissawm/11template) is still pointing to a Hack MD file, shall we publish those in the GitBook as well?

### PR DESCRIPTION
**Pull request by Cheukting**
@melissawm this is to fix the still hackmd link of our page

closes #27

### CODE REVIEW

**Comment by Cheukting** [comment]
Please check if I am doing it properly, I am new to Jupyter book

File: unconference/_toc.yml
Comment (OriolAbril) [inline unconference/_toc.yml:9]:
```suggestion
  - file: mentoring_guide_projects.md
    sections:
    - file: mentoring_onboarding_checklist.md
    - file: mentoring_meeting_template.md
```

I think they'd fit better as sections instead of chapters. That is, they won't appear directly on the left sidebar, instead the mentoring section will have an arrow to "unfold" itself and reveal its sections. The unfolding will also happen automatically whenever one is reading either of the 3 documents. Reference and example: https://jupyterbook.org/en/stable/structure/toc.html#use-chapter-sub-sections

**Review by OriolAbril — COMMENTED** [review]
I will try to check the preview locally. I am not 100% sure what will happen with the links as currently written. I think we want them to be links to another page on the website (and quite sure they are) but sometimes they can be interpreted as downloadable files instead

File: unconference/_toc.yml
Comment (Cheukting) [reply unconference/_toc.yml:9]: Good call @OriolAbril, I wanted it to be sessions and didn't know the way, let's merge your suggestions

File: unconference/mentoring_meeting_template.md
Comment (OriolAbril) [inline unconference/mentoring_meeting_template.md:4]:
```suggestion
## Personal update
```

File: unconference/mentoring_meeting_template.md
Comment (OriolAbril) [inline unconference/mentoring_meeting_template.md:13]:
```suggestion
## Ongoing project(s)
```

File: unconference/mentoring_meeting_template.md
Comment (OriolAbril) [inline unconference/mentoring_meeting_template.md:25]:
```suggestion
## Career guidance/expectations
```

File: unconference/mentoring_meeting_template.md
Comment (OriolAbril) [inline unconference/mentoring_meeting_template.md:34]:
```suggestion
## Wrapping up
```

**Review by OriolAbril — COMMENTED** [review]
I checked the preview locally and the sections are working as intended. Sphinx did complain about the headings though. Heading hierarchy must decrease 1 by 1, otherwise sphinx makes them decrease 1 by 1. Currenly it prints a warning and proceeds to use `h2` for all the headings in the meeting template, I added suggestions below

**Review by melissawm — APPROVED** [review]
Thanks, @Cheukting ,  looks great to me!

**Comment by OriolAbril** [comment]
@melissawm I think so far you are the only one with commit rights so you'll have to merge this and the other pending PRs

**Pull request merged**
